identity/manager_test.go

```diff
@@ -89,3 +89,66 @@ func Test_HasIdentity(t *testing.T) {
 	assert.True(t, manager.HasIdentity("0x000000000000000000000000000000000000000a"))
 	assert.False(t, manager.HasIdentity("0x000000000000000000000000000000000000000B"))
 }
+
+func Test_IsUnlocked(t *testing.T) {
+	ks := getIdentityTestKeystore()
+
+	tests := []struct {
+		Address    string
+		Passphrase string
+	}{
+		{
+			"0x53a835143c0eF3bBCBFa796D7EB738CA7dd28f68",
+			"",
+		},
+		{
+			"0x1e35193c8cadAA15b43B05ae3D882C91F49BB0Aa",
+			"test_passphrase",
+		},
+	}
+
+	// our test identity
+	manager := NewIdentityManager(ks)
+	for _, test := range tests {
+		identity := FromAddress(test.Address)
+
+		assert.False(t, manager.IsUnlocked(identity))
+
+		err := manager.Unlock(identity, test.Passphrase)
+
+		assert.Nil(t, err)
+		assert.True(t, manager.IsUnlocked(identity))
+	}
+}
+
+func Test_UnlockError(t *testing.T) {
+	ks := getIdentityTestKeystore()
+	testData := struct {
+		Address    string
+		Passphrase string
+	}{
+		"0x53a835143c0eF3bBCBFa796D7EB738CA7dd28f68",
+		"123",
+	}
```

Comment:
Is it a common practice to define test data struct even in a single case test?

Reply:
Probably unnecessary:)